src/lib/components/FieldInput.svelte

```diff
@@ -1,9 +1,10 @@
 <script lang="ts">
 	import Field from './Field.svelte';
+	import type { LocalizedString } from 'typesafe-i18n';
 
 	export let name: string;
-	export let label: string;
-	export let placeholder: string | undefined = undefined;
+	export let label: LocalizedString;
+	export let placeholder: LocalizedString | string | undefined = undefined;
```

Comment:
We now have a dedicated `LocalizedString` type, so unless we are expecting a random `string` we should use that type by default. The good thing is that the linter will yell at you if you do it wrong 😅